File: EjerciciosJavaKotlinHLC2/Java/src/Anime.java
public class Anime {
    String nombre;
    int episodios ;
    String género;

    public Anime(String nombre, int episodios, String género) {
        this.nombre = nombre;
        this.episodios = episodios;
        this.género = género;
    }

    @Override
    public String toString() {
        return "Anime{" +
                "nombre='" + nombre + '\'' +
                ", episodios=" + episodios +
                ", género='" + género + '\'' +
                '}';
    }
}

File: EjerciciosJavaKotlinHLC2/Kotlin/src/Anime.kt
class Anime(var nombre: String, var episodios: Int, var género: String) {
    override fun toString(): String {
        return "Anime{" +
                "nombre='" + nombre + '\'' +
                ", episodios=" + episodios +
                ", género='" + género + '\'' +
                '}'
    }
}